pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -133,10 +148,20 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  # https://github.com/LostRuins/koboldcpp/blob/0fc18d2d82633d107a85c5ba725f665c8228e1c7/Makefile#L136
+  # https://github.com/LostRuins/koboldcpp/blob/0fc18d2d82633d107a85c5ba725f665c8228e1c7/Makefile#L73C1-L73C69
+  # Remove `warning: argument unused during compilation: '-s' [-Wunused-command-line-argument]` on Darwin
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-warn "-march=native" "" \
+      --replace-warn "-mtune=native" "" \
+      --replace-warn " -s " " "
```

Comment:
TIL: https://github.com/NixOS/nixpkgs/blob/4b6a0890371976e066083789f3039c6f78af95d6/pkgs/build-support/cc-wrapper/cc-wrapper.sh#L121-L134

But maybe it doesn't hurt to keep this

Reply:
That's really handy. Well, I was looking at the log with `NIX_DEBUG`, and indeed it removes `native` automatically. So, patching the Makefile for that is pointless, although it is confusing since the print from KoboldCpp is misleading.